packages/nimble-components/.storybook/transformSource.js

```diff
@@ -1,5 +1,4 @@
-import prettier from 'prettier/standalone';
-import parserHTML from 'prettier/plugins/html';
+const beautifyHTML = require('js-beautify').html;
```

Comment:
Would be nice to have esm support: https://github.com/beautifier/js-beautify/issues/2199

But not a source dependency so not as big a deal breaker. Let's go!